templates/base/eslintrc

```diff
@@ -1,9 +1,19 @@
-{
-     "env": {
+{    "ecmaFeatures" : {
+        "modules" : true
+                      },
```

Comment:
this should be indented to match the beginning of the "ecmaFeatures" line above... sorry just noticed this when I pulled it down